TheMET/TheMET/Services/FavoritesService.swift

```diff
@@ -36,23 +38,29 @@ class FavoritesService {
                 self?.favoriteArts = arts
```

Comment:
This loading also changes the `favoriteArts` and must cause a notification to be posted.